package.json

```diff
@@ -29,6 +29,7 @@
     "jsdom": "^9.6.0",
     "less": "^2.2.0",
     "less-loader": "^2.0.0",
+    "lodash.isnil": "^4.0.0",
```

Comment:
I think I'll just change this to lodash and use lodash/isNil below. I tested the build and compressed file size didn't change, uncompressed was lower by a couple of bytes